frontend/src/diagram/palette/ContextualPaletteContainer.tsx

```diff
@@ -0,0 +1,88 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2020 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import React, { CSSProperties } from 'react';
+import PropTypes from 'prop-types';
+import { EditLabelAction } from 'sprotty';
```

Comment:
Does this container need a dependency to Sprotty?

Reply:
I could reply the same thing https://github.com/eclipse-sirius/sirius-components/pull/143#discussion_r534007046